media-libs/libmpdclient/libmpdclient-2.14.recipe

```diff
@@ -2,27 +2,30 @@ SUMMARY="A library for interfacing Music Player Daemon"
 DESCRIPTION="Music Player Daemon (MPD) is a flexible, powerful, server-side application \
 for playing music. Through plugins and libraries it can play a variety \
 of sound files while being controlled by its network protocol."
-HOMEPAGE="http://www.musicpd.org/"
-COPYRIGHT="2003-2017 The Music Player Daemon Project"
+HOMEPAGE="https://www.musicpd.org/"
+COPYRIGHT="2003-2018 The Music Player Daemon Project"
 LICENSE="BSD (3-clause)"
 REVISION="1"
 SOURCE_URI="https://github.com/MusicPlayerDaemon/libmpdclient/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="e1583fbcae89ad543a6cc44caeaf16b31e709831f5b7eac4413f78c54a21db95"
+CHECKSUM_SHA256="262f1ff13e6ee11cca2ef93f2eda10ec6953a7b8e628572645e707fe530fbbb6"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
```

Comment:
```diff
 CHECKSUM_SHA256="262f1ff13e6ee11cca2ef93f2eda10ec6953a7b8e628572645e707fe530fbbb6" 
+SOURCE_FILENAME="libmpdclient-$portVersion.tar.gz"

 ARCHITECTURES="!x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
```

Reply:
The name of the upstream tarball is `v2.13.tar.gz` so I'd recommend adding a [`SOURCE_FILENAME`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name) as suggested in my previous comment.
Regarding the x86 arch, I have tested it (before writing my previous comment) so you can mark it as tested if you wish.